### DISCUSSION TITLE
How to call function with parameters ?

### DISCUSSION THREAD

**Discussion by fadi-legit** [maizzle/framework#601] [Q&A]
im not sure how to use functions with parameters ? is it supported ?
e.g. : i want function that return dynamic value based on input
var plus = (a,b) => a+b

i was expected this usage:

**in config file:**
`module.exports = {
plus = (a,b) => a + b
}`

**in the HTML i will use it this way:**
```
<each loop="item, index in numbers">
 <div>{{item.a}} + {{item.b}} = {{ plus(item.a, item.b) }}</div>
</each>
```

**Comment by cossssmin** [comment]
That should work, just that everything except events are referenced off of the `page` object:

```diff
<each loop="item, index in numbers">
- <div>{{item.a}} + {{item.b}} = {{ plus(item.a, item.b) }}</div>
+ <div>{{item.a}} + {{item.b}} = {{ page.plus(item.a, item.b) }}</div>
</each>
```

Docs: https://maizzle.com/docs/functions/#functions

**Reply by DavidPetrasek** [reply]
I tried it according to the docs, but it does work only without parameters. I also tried defining my function with arguments inside script block inside a component and it also didn't work.

**Reply by DavidPetrasek** [reply]
This could be a new feature.
